hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -381,6 +373,7 @@ profile::jenkinscontroller::jcasc:
               - docker-highmem
             useAsMuchAsPosible: false
             spot: false
+            userData: *userData
```

Comment:
```suggestion
            userData: *ec2DatadogInitScript
```

Reply:
using initscript could be confusing as it's the main difference between azure and amazon, one use the userData and the other initscript ... but I agree that the variable could be more efficient with the provider name in

Reply:
Note the prefix in front of these 2 variable names.
Also these blocks are init scripts no?

Reply:
Do you agree on these items?

- The initial name should be changed (we all agree on this :) )
- prefixing with "ec2" is legit, but the code is not relative to EC2. It's the fact that this "common" code is passed as user-data with a shebang on the first line (ref. cloud init documentation).
- "init script" has different meaning and can be confusing:
  - Init script is executed when the agent is started, through the SSH command. Azure can run it as root, but not EC2.
- The wording "init" seems to be related to "initialization of datadog". Proposal: "bootstrap" (which would implie both the "initial setup" and "starting the service".
- It' seems nice to use "script" as it implies that the pointed data is .. a (shell) script